CHANGELOG.md

```diff
@@ -4,6 +4,23 @@
 <!--- Distributed under the Boost Software License, Version 1.0. (See accompanying -->
 <!--- file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt) -->
 
+## 0.17.0 (2023-08-02)
+
+### New features
+
+- Add `pika::wait` to wait for the runtime to be idle. ([#704](https://github.com/pika-org/pika/pull/704))
+- The debugger can now be attached after printing failure information. Previously the debugger could be attached before printing the error information. ([#712](https://github.com/pika-org/pika/pull/712))
+- `handle_print_bind` now also prints the thread pool when a mask is empty to be consistent to the printing when a mask is set. ([#710](https://github.com/pika-org/pika/pull/710))
```

Comment:
Please refer to `--pika:print-bind` rather than `handle_print_bind` since the latter is just an internal helper, and the former is the actual option a user would use to print pool/binding information.

Reply:
E.g.
```suggestion
- `--pika:print-bind` now also prints the thread pool of a thread when thread binding is disabled to be consistent with the output when thread binding is enabled. ([#710](https://github.com/pika-org/pika/pull/710))
```